contracts/libraries/ChainId.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.19;
+
+/// @title Function for getting the current chain ID
+library ChainId {
+    /// @dev Gets the current chain ID
+    /// @return chainId The current chain ID
+    function get() internal view returns (uint256 chainId) {
```

Comment:
changed `pure` to `view`

Reply:
oo in solidity 0.8 we now have `block.chainid` so this lib is no longer useful